packages/databricks-vscode/src/cluster/ClusterManager.ts

```diff
@@ -1,7 +1,7 @@
 import {compute, Time, TimeUnits} from "@databricks/databricks-sdk";
 import {Cluster} from "../sdk-extensions";
 import {CancellationTokenSource, Disposable} from "vscode";
-
+import _ from "lodash";
```

Comment:
Let's add `lodash` explicitly to our package.json, otherwise builds can break when we update dependencies. We can also "optimise" and add only a dependency on `lodash.isEqual`